packages/kit/src/views/Setting/pages/CustomNetwork/index.tsx

```
@@ -64,7 +64,7 @@ function AddCustomNetwork() {
     blockExplorerUrl: routeBlockExplorerUrl,
   } = route.params ?? {};
 
-  const isEditMode = !!route.params.chainId;
+  const isEditMode = !!(route.params ?? {}).chainId;
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)